### ISSUE TITLE
cli: exit with code when some error happened.

### ISSUE THREAD

**Issue by Txiaozhe** [alibaba/pipcook#163]
such as:
```js
export const start: StartHandler = async (filename: string) => {
  if (!filename) {
    spinner.fail('Please specify the config path');
    return;
  }

  if (!existsSync(filename)) {
    spinner.fail(`${filename} not exists`);
     return;
  }
```
return directly will make ci green although err was happened.

**Comment by yorkie** [comment]
Yea, we should correct the exit code always, even though we don't throw an error. This causes the build on #137 is green on failures.

**Comment by Txiaozhe** [comment]
This issue should be closed because of #164 and #162 has been merged:)
